### ISSUE TITLE
Can't update BIOS for Lenovo ThinkCentre desktops

### ISSUE THREAD

**Issue by athmanb** [Zwable/BIOS-Update-for-Enterprise#2]
In ThinkPads, if you query the BIOS version with "(Get-WmiObject Win32_Bios).SMBIOSBIOSVersion", you get a value like "N30ET32W (1.15 )". This is correctly parsed by the script and the BIOS update proceeds.

But, in ThinkCentre desktops (i.e. the ThinkCentre M920s), if you query "(Get-WmiObject Win32_Bios).SMBIOSBIOSVersion", the output value is only "M1UKT59A" without any followup parentheses and version.

This leads to the script outputting:
```
Current installed version is '' and the available one is ''
Current installed version is compliant or higher than the available one, exiting...
```

and never applying an update even if one is technically required.

**Comment by Zwable** [comment]
This is now fixed using another WMI class more suitable for all vendors. Tested on ThinkPad P51s W10DG

**Comment by Zwable** [comment]
@athmanb can you confirm that it fixed the problem in your end?

**Comment by athmanb** [comment]
I have quickly tested it on two notebook systems that are already up to date, and it works correctly there:

```
Running BIOS upgrade for the model 'ThinkPad T470s W10DG'
Current installed version is '1.40' and the available one is '1.40'
Current installed version is compliant or higher than the available one, exiting...
```

I'll test it on other systems that have yet to be updated but it'll be a few days because I need confirmation by the customer/users first.
